### PR TITLE
feat(checkout): CHECKOUT-5682 Add documentation for new Checkouts/settings endpoint

### PR DESCRIPTION
**Pull request by valentindellangela**
## What
Adding documentation for the newly created V3 Checkouts/settings endpoint

## Why
This endpoint will be used by headless merchants to specify the url of their custom checkout script / order confirmation script.

## Test
Swagger IO:
image

Review @bigcommerce/checkout  @traci707

### CODE REVIEW

**Comment by valentindellangela** [comment]
@traci707  Do you know how to fix those issues - they are not part of my changes.
Example:
> line 957, col 15, `default` property type must be array (oas3-valid-schema-example) at path #/components/schemas/CartCreatePostData/properties/line_items/oneOf/0

**Comment by traci707** [comment]
Hi @valentindellangela I learned that many spec files were changed to OAS3 because of the launch of MSF.  These are not real errors.  I was told that we can merge these PRs.

**Review by bc-shawnwang — COMMENTED** [review]
should we add some comments for this behavior?

When order_confirmation_use_custom_checkout_script=true we default custom_order_confirmation_script_url to empty string

File: reference/checkouts.v3.yml [hunk 1]
```diff
@@ -1654,6 +1674,29 @@ definitions:
...
+  CheckoutsSettingsRequest:
+    title: Checkouts settings request
+    type: object
+    properties:
+      custom_checkout_script_url:
+        type: string
+        description: Custom checkout script url to replace our default checkout.
+      order_confirmation_use_custom_checkout_script:
+        type: boolean
+        description: When order_confirmation_use_custom_checkout_script=true we default custom_order_confirmation_script_url to empty string.
```
Comment (bc-shawnwang) [inline reference/checkouts.v3.yml:1696]: 👍

**Review by bc-shawnwang — APPROVED** [review]
Looks good.

**Review by bc-shawnwang — APPROVED** [review]

**Pull request closed without merging**